.github/dependabot.yml

```diff
@@ -22,14 +23,30 @@ updates:
     - kind/enhancement
     - release-note/misc
 
+  - package-ecosystem: gomod
+    directory: /
+    schedule:
+      interval: monthly
+    open-pull-requests-limit: 15
+    rebase-strategy: "disabled"
+    allow:
+        # cloud provider SDKs are updated too frequently, update them monthly.
+      - dependency-name: "github.com/aliyun/alibaba-cloud-sdk-go"
+      - dependency-name: "github.com/aws/*"
+      - dependency-name: "github.com/Azure/*"
+    labels:
+    - kind/enhancement
+    - release-note/misc
+
 # # # # # # # # # # # # # # # #
 #        GitHub Actions       #
 # # # # # # # # # # # # # # # #
 
   - package-ecosystem: github-actions
     directory: /
     schedule:
-      interval: daily
+      interval: weekly
+      day: sunday
     open-pull-requests-limit: 5
```

Comment:
I would suggest removing the limit if we do it on weekly basis (or bumping the limit) as we probably don't want to end up in a situation where dependabot isn't able to catch up.